woocommerce/woocommerce
# Remove the "We're here to help" spotlight

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #43300 . 

Previously a help modal would appear when a user visits a task for the second time while that task is incomplete. This wasn't as helpful as it seemed, because it would interfere with other questions being asked of the user.

<img width="1792" alt="help modal" src="https://github.com/user-attachments/assets/bb425410-e957-4047-8a4f-ac05e05c6f86">

I am wondering whether to also remove the associated user meta for keeping state from the database? It could perhaps be re-purposed in the future for a similar modal. On the other hand, it feels better to pick up the trash and leave things cleaner. I think it could be tackled in a follow-up if we want to go a head with it.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to WooCommerce > Home
2. Click on Products task
3. Return back to home
4. Click on Products task
5. See that a help modal does not appear



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
